README.md

```diff
@@ -89,6 +89,145 @@ refresh_token = jwt_response[REFRESH_SESSION_TOKEN_NAME].get("jwt")
 
 The session and refresh JWTs should be returned to the caller, and passed with every request in the session. Read more on [session validation](#session-validation)
 
+### Enchanted Link
+
+Send an email containing three numbered links to a user.
+This method is similar to [Magic Link](#magic-link) but differs in two major ways:
```

Comment:
maybe we can improve a bit wording on this
@gshriki @rishibhargava any chance you have some text on the enchanted link vs. magic link